### ISSUE TITLE
[4.4 dev6 - Windows] All button signals are broken in exported exe

### ISSUE THREAD

**Issue by jupiterbjy** [godotengine/godot#100297]
### Tested versions

(All win 11 forward+)

**When creating & exporting new project**
- Reproducible in Godot 4.4 dev 6
- Works fine for same MRE in Godot 4.4 dev 5

<br>

**When loading & exporting projects from dev 5 -> 6**
- Inconsistently reproducible, more testing needed to triangulate cause
- All affected projects broken in dev 6 remains broken when loaded back in Godot 4.4 dev 5

---

### System information

Godot Engine v4.4.dev6.official.1f47e4c4e - Vulkan 1.3.280 - Forward+ - Using Device #0: NVIDIA - NVIDIA GeForce RTX 4070 Ti SUPER

---

### Issue description

All button's signals cuts off w/o any error on console in dev6 w/ windows export.

Once project is loaded into dev6, such behavior persists back in dev5.

Workaround I found to fix project back in dev5 is following:

for **_EVERY_** scene that **_OWNS_** buttons (not inherited one, not instanced one) do:

1. force changes to scene (i.e. adding pointless `Control` node in scene's root)
2. save, export and verify it works. (If not try reloading/refreshing project)
3. remove 1st step's change and re-export.

This seems to fix simple scenes or inherited ones but instanced buttons seems to be troublesome.

---

### Steps to reproduce

1. Create button
2. Connect signals
3. Export for x86_64 window
4. All button signals are broken

or

1. Load older version's projects
3. Export for x86_64 window
4. All button signals are broken

---

### Minimal reproduction project (MRP)

MRE: [mre_44_dev5_6.zip](https://github.com/user-attachments/files/18104452/mre_44_dev5_6.zip)

<br>

Entire MRE source code & structure for reproducing without MRE just in case:

![Image](https://github.com/user-attachments/assets/c9b34d4d-3f34-406b-8174-6b1a868f2e4c)

```gdscript
extends Control


@onready var state_label: Label = %ButtonStateLabel
@onready var count_label: Label = %PressCountLabel
var count: int = 0


func _on_button_button_down() -> void:
	self.state_label.text = "Button State: DOWN"


func _on_button_button_up() -> void:
	self.state_label.text = "Button State: UP"


func _on_button_pressed() -> void:
	count += 1
	self.count_label.text = "Press Count: %s" % count
```

<br>

Video of MRE demonstration
https://github.com/user-attachments/assets/f5982d15-860c-4ebe-8c16-75bbf9585079

**Comment by genomcnew** [comment]
I think I'm having a similar issue when exporting to android.
Buttons are being pressed but no signals are triggering.

Only able to reproduce on dev 6. No issues with dev 5 or 4.3 stable.

**Comment by AeioMuch** [comment]
Duplicate of https://github.com/godotengine/godot/issues/100097
It should be fixed already by https://github.com/godotengine/godot/pull/100235

**Comment by jupiterbjy** [comment]
@AeioMuch darn, I tried deleting `.godot` but just let it regen in dev6, how come I didn't think of regenerating it in dev5! Brilliant solution. I was doing some stupid workaround meanwhile..

I think it might be better to rename #100097 since it also happen for already existing, non-dev6 `.godot` dir! - Would've help finding issue as many would be opening existing project(hopefully after backup or VSC) and encounter this issue, hence with such context search keyword lands nothing near that issue's title. Or I just did real bad job at searching haha

Closing this!
